orbs/phraseapp/.phraseapp-sample.yml

```diff
@@ -0,0 +1,12 @@
+# Example of configuration file
```

Comment:
Phraseapp changed name to Phrase a while ago. While we still use phraseapp in informal conversation, we might want to use the real name in the folders. They also support `.phrase.yml`

Reply:
Also maybe use the `.phrase.sample.yml` name already used in jobteaser, ui-jobteaser, notification, etc.